scripts/dev

```diff
@@ -1,5 +1,10 @@
 #!/bin/bash
 
+set -e
+cd $(dirname $0)/..
+
+./scripts/postinstall
```

Comment:
There’s got to be a better way to do this `postinstall`. I’m going to try moving all of the GraphiQL dependencies into the project root and see how that goes. Makes it harder to extract `graphiql` into its own package eventually, but it would avoid some of these workarounds.